django/templatetags/i18n.py

```diff
@@ -127,9 +127,7 @@ def render(self, context, nested=False):
             message_context = self.message_context.resolve(context)
         else:
             message_context = None
-        tmp_context = {}
-        for var, val in self.extra_context.items():
-            tmp_context[var] = val.resolve(context)
+        tmp_context = {var: val.resolve(context) for (var, val) in self.extra_context.items()}
```

Comment:
You don't need the parentheses around `var, val`. In fact this line could also be merged below:
```python
         context.update({var: val.resolve(context) for var, val in self.extra_context.items()})
```